code/go/0chain.net/conductor/conductor/executor.go

```diff
@@ -301,6 +301,11 @@ func (r *Runner) WaitShareSignsOrShares(ssos config.WaitShareSignsOrShares,
 }
 
 func (r *Runner) WaitAdd(wadd config.WaitAdd, tm time.Duration) (err error) {
+	// filter initialized nodes
+	// nodes := r.server.Nodes()
+	// for name := range nodes {
+	// 	wadd.Take(name)
+	// }
```

Comment:
Remove commented code if not used

Reply:
fixed in [dd64987](https://github.com/0chain/0chain/pull/1851/commits/dd64987f770e7daad5607ab8a72413cc69a5b5bb)